sdk/src/main/java/com/dailymotion/android/player/sdk/PlayerWebView.java

```diff
@@ -378,6 +447,11 @@ private void handleEvent(String e) {
             mEventListener.onEvent(event, map);
         }
 
+        /* Expose via POJO only event we are supporting */
```

Comment:
Wording: use "Only expose the POJO events we are supporting"  instead ?

Reply:
👍